packages/utils-dom/src/htmlelement.ts

```diff
@@ -206,6 +206,26 @@ export class HTMLElement extends Node {
         return null;
     }
 
+    /**
+     * Non-standard.
+     * Zero-based location of the specified attribute in original source code.
+     *
+     * @param name The name of the attribute to get the location for.
+     */
+    public getAttributeLocation(name: string): ProblemLocation {
```

Comment:
What is an attribute in this context? An element's attribute?

Reply:
Yes, an HTML attribute on an element.